scripts/devenv.sh

```diff
@@ -9,6 +9,7 @@ docker run -it \
 	--privileged \
 	-v /var/run/docker.sock:/var/run/docker.sock \
 	-v `pwd`:/gopath/src/github.com/Azure/acs-engine \
+	-v ~/.azure:/root/.azure \
```

Comment:
Interesting, looks like this happens when running `devenv.sh` with `sudo` and not added to docker group.

Just pushed a change to make sure it's running as user, let me know if this works.